GitUp/Application/AppDelegate.m

```diff
@@ -411,6 +413,11 @@ - (void)applicationDidFinishLaunching:(NSNotification*)notification {
 #endif
 }
 
+- (void)_getUrl:(NSAppleEventDescriptor*)event withReplyEvent:(NSAppleEventDescriptor*)replyEvent {
+  NSString* url = [[event paramDescriptorForKeyword:keyDirectObject].stringValue stringByReplacingOccurrencesOfString:@"github-mac://openRepo/" withString:@""];
```

Comment:
This doesn't seem very clean: what's the exact string passed retrieved from the event?